src/model/payload.js

```diff
@@ -0,0 +1,6 @@
+export default class payload {
```

Comment:
Name convention is class names should be camel case, each word should start with an uppercase letter. This comment applies to all classes in this PR.

I appreciate this may be inherited from Andrew's prototype.

Reply:
Sure, I was following his conventions, do you prefer we change?